nautobot_design_builder/design.py

```diff
@@ -563,6 +566,9 @@ def _load_instance(self):  # pylint: disable=too-many-branches
                     field_values[query_param] = model
             try:
                 self.instance = self.relationship_manager.get(**query_filter)
+                if self.environment.import_mode and self.metadata.action != ModelMetadata.UPDATE:
+                    self.metadata.attributes.update(field_values)
+
```

Comment:
What are the circumstances that would cause this block of code to be executed? If the builder is not running in import mode, then this block is reached from a `create_or_update` when the requested object already exists and will now be updated. With this change I think we essentially treat all `create` actions as `create_or_update` when in import mode, but I doubt that there is even a query filter for most cases of `create`. If a design is intended to be able to be run in `import` mode then there shouldn't be any actions that are create only, they should be `create_or_update` in order to identify the query fields.